yeonjae/programmers-0/최빈값구하기.js

```diff
@@ -0,0 +1,12 @@
+function solution(array) {
+  const obj = {};
+
+  for (const i in array) {
+    array[i] in obj ? (obj[array[i]] += 1) : (obj[array[i]] = 1);
+  }
+
+  const mostTimes = Math.max(...Object.values(obj));
+  const mosFrequent = Object.keys(obj).filter((k) => obj[k] == mostTimes);
+
+  return mosFrequent.length == 1 ? +mosFrequent[0] : -1;
+}
```

Comment:
```suggestion
function solution(array) {
  const obj = array.reduce((acc, cur) => {
    acc[cur] = (acc[cur] || 0) + 1;
    return acc;
  }, {});

  const mostTimes = Math.max(...Object.values(obj));
  const mosFrequent = Object.keys(obj).filter((k) => obj[k] == mostTimes);

  return mosFrequent.length == 1 ? +mosFrequent[0] : -1;
}
```
- 풀이 잘봤습니다~~ㅎㅎ 배열을 for...in문으로 순회해주셨는데 차라리 순회를 목적이라면 배열이기 때문에 배열 메서드인 reduce나 forEach를 활용하는게 더 적절할 것 같습니다 😄 고생하셨씁니다